default/AI/ShipDesignAI.py

```diff
@@ -527,6 +535,8 @@ def _update_buildable_items_this_turn(self, verbose=False):
                     if res:
                         print "Success: Added Test Design %s, with result %d" % (testdesign_name, res)
                         testdesign_names_part.append(testdesign_name)
+                        # update cache since we just created a new design in fo
+                        _ = _get_design_by_name(testdesign_name,update_invalid=True)
```

Comment:
As we basically use a getter function here to update the cache, I think the ```_ = ``` is justified to explicitly state we do not care about the return value and that the call is intentional.

However, I wouldn't mind removing it as long as the comment is clear enough...